index.html

```diff
@@ -79,7 +79,7 @@
       <div class="container">
         <br><br>
          <div class="container" id="Welcome">
-        <h1 class="header center orange-text">Humblefool Cup</h1>
+        <a href="humblefool/humblefool.html"><h1 class="header center orange-text">Humblefool Cup</h1></a>
```

Comment:
i dont think we should link the events name to the tribute page. we should link the tribute page when harsha sirs name comes up only